Dockerfile.buildx

```
@@ -1,17 +1,16 @@
-FROM --platform=$BUILDPLATFORM golang:1.18 AS deps
+FROM --platform=$BUILDPLATFORM golang:1.22 AS build
 ARG TARGETPLATFORM
 ARG BUILDPLATFORM
 
 COPY . /src
 WORKDIR /src
-RUN go get -v ./...
 RUN go vet -v ./...
 RUN CGO_ENABLED=0 GO111MODULE=on go build
```

If we would like to see this in Dynatrace in detail:
```suggestion
RUN go build -ldflags "-linkmode=external "
```

but in this case, it cannot run in `scratch` anymore.

---

Will handle it in our separate wrapper